unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/CharacterPreview/PreviewCameraZoom.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using UnityEngine;
+using UnityEngine.EventSystems;
+
+namespace MainScripts.DCL.Controllers.HUD.CharacterPreview
+{
+    public class PreviewCameraZoom : MonoBehaviour, IPointerEnterHandler, IPointerExitHandler
+    {
+        public event Action<Vector3> OnZoom;
+
+        [SerializeField] private float zoomSpeed = 2.0f;
+        [SerializeField] private float smoothTime = 0.2f;
+
+        private bool isFocused;
+        private Vector3 currentZoomDelta;
+        private Vector3 currentZoomVelocity;
+
+        public void OnPointerEnter(PointerEventData eventData) =>
+            isFocused = true;
+
+        public void OnPointerExit(PointerEventData eventData)
+        {
+            isFocused = false;
+            currentZoomDelta = Vector3.zero;
+        }
+
+        private void Update()
```

Comment:
Dont use Update, it has lot of overhead. Use a task (with cancellation when is not focused) instead.

Reply:
Done: https://github.com/decentraland/unity-renderer/pull/5162/commits/86d12827d8de43b39c5a5873355c3f7e5c1e43ed